sphinx_gherkindoc/writer.py

```diff
@@ -303,11 +299,12 @@ def steps(steps: List[behave.model.Step], step_format: str = "{}") -> None:
                 text(step.text)
 
     def examples(
-        scenario: behave.model.Scenario, feature: behave.model.Feature
+        scenario: Optional[behave.model.Scenario], feature: behave.model.Feature
     ) -> None:
-        for example in getattr(scenario, "examples", []):
+        example_source = scenario if scenario else feature
```

Comment:
I think it makes sense to just pass in `example_source` as a param instead of scenario. So in one case, you'll pass in the feature object as both params. Then you don't have to do the filter below.